docs/topics/request-response.rst

```diff
@@ -363,26 +363,28 @@ are some special keys recognized by Scrapy and its built-in extensions.
 
 Those are:
 
-* :reqmeta:`dont_redirect`
-* :reqmeta:`dont_retry`
-* :reqmeta:`handle_httpstatus_list`
-* :reqmeta:`handle_httpstatus_all`
-* :reqmeta:`dont_merge_cookies`
+* :reqmeta:`bindaddress`
 * :reqmeta:`cookiejar`
 * :reqmeta:`dont_cache`
-* :reqmeta:`redirect_reasons`
-* :reqmeta:`redirect_urls`
-* :reqmeta:`bindaddress`
+* :reqmeta:`dont_merge_cookies`
 * :reqmeta:`dont_obey_robotstxt`
-* :reqmeta:`download_timeout`
-* :reqmeta:`download_maxsize`
-* :reqmeta:`download_latency`
+* :reqmeta:`dont_redirect`
+* :reqmeta:`dont_retry`
 * :reqmeta:`download_fail_on_dataloss`
-* :reqmeta:`proxy`
-* ``ftp_user`` (See :setting:`FTP_USER` for more info)
+* :reqmeta:`download_latency`
+* :reqmeta:`download_maxsize`
+* :reqmeta:`download_timeout`
 * ``ftp_password`` (See :setting:`FTP_PASSWORD` for more info)
-* :reqmeta:`referrer_policy`
+* ``ftp_user`` (See :setting:`FTP_USER` for more info)
+* :reqmeta:`handle_httpstatus_all`
+* :reqmeta:`handle_httpstatus_list`
 * :reqmeta:`max_retry_times`
+* :reqmeta:`proxy`
+* :reqmeta:`redirect_reasons`
+* :reqmeta:`redirect_urls`
+* :reqmeta:`referrer_policy`
+
+
 
```

Comment:
Could you leave a single empty line here? Those 2 new empty lines do not seem necessary.